Libbna/CUSTOM-CMS
# #213:  Add search icon to navbar.

* Added search icon and input field to navbar.
* Also solved the error ocurring on `localhost/custom_cms/` root route.
* The root route is now getting redirected to home page.
* Steps for testing : 
   * The navbar should contain the search input field and button.
   * Go to `localhost/custom_cms/` root route.
   * The root route should get redirected to `/home` route automatically.

* Refer screenshot for search icon and input field.

![Screenshot from 2022-01-13 15-55-34](https://user-images.githubusercontent.com/83503327/149312851-bedccea3-fa42-4a11-8bb0-3f488c1e344c.png)
